addons/pad_project/models/project.py

```diff
@@ -11,6 +11,20 @@ class ProjectTask(models.Model):
     description_pad = fields.Char('Pad URL', pad_content_field='description')
     use_pad = fields.Boolean(related="project_id.use_pads", string="Use collaborative pad")
 
+    @api.model
+    def create(self, vals):
+        # When using quick create, the project_id is in the context, not in the vals
+        project_id = vals.get('project_id', False) or self.env.context.get('active_id', False)
```

Comment:
I would call default_get and/or add missing default value if project_id is not in vals, otherwise LGTM